### PR TITLE
Blog app - controllers

### PR DESCRIPTION
**Pull request by tumainimaganiko**
In this project, I did the following:

- Use controllers to handle requests and render empty views.
-  For each URL I created:
    - A route.
    - An action in the correct controller.
    - A view file.

-  View files include only placeholders written in plain HTML, e.g:
    - Here is a list of posts for a given user
    
-  I added code to handle **only** the 4 required URLs.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :users, only: [:index, :show]
+
+  resources :users do
+    resources :posts, only: [:index, :show]
```
Comment (nahnahdev) [inline config/routes.rb]: - [ ] Great job so far @tumainimaganiko The implementation you've provided seems to be mostly correct for defining routes in a Ruby on Rails application. You're defining routes for users and their associated posts. :+1:
- [ ] However, there's a small issue related to redundancy in your route definition. You have defined the `resources :users` block twice. You should only define it once and nest the` posts resources` inside that block. The corrected version of your code can look something like this:

```ruby
Rails.application.routes.draw do
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
end

```

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @tumainimaganiko , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :fire: 
- Linters well setup and passing :heavy_check_mark: 
- Professional README :heavy_check_mark: 
- `rails generate scaffold` command was not used :clinking_glasses: 
- Has `users_controller` and `posts_controller` with `#index` and `#show` actions :heavy_check_mark: 
- Has corresponding view files for each of the controller actions :heavy_check_mark: 
- Has `users` and `posts` routes added to `config/routes.rb` :heavy_check_mark: 
- Maintained a RESTful URL :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (tumainimaganiko) [reply config/routes.rb]: Thanks @NahnahAJ for noticing that bug. I have kindly implemented it as suggested

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @tumainimaganiko,

✔️ Implementation of nested routes

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**